python/kfserving/kfserving/storage.py

```diff
@@ -193,9 +193,10 @@ def _download_local(uri, out_dir=None):
     @staticmethod
     def _create_minio_client():
         # Remove possible http scheme for Minio
-        url = re.compile(r"https?://")
-        minioClient = Minio(url.sub("", os.getenv("S3_ENDPOINT", "")),
+        url = urlparse(os.getenv("S3_ENDPOINT", ""))
```

Comment:
`S3_ENDPOINT` does not contain the scheme, it is actually `AWS_ENDPOINT_URL`(https://github.com/kubeflow/kfserving/blob/master/pkg/controller/kfservice/resources/credentials/s3/s3_secret.go#L118)

Reply:
thanks! fixed